tree-sitter-python/test/Main.hs

```diff
@@ -31,5 +33,6 @@ prop_simpleExamples = property $ do
   "pass" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right pass] }
   "1" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right one] }
   "expensive" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right function] }
+  "1\npass" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right one, Right pass] }
 
-main = void $ checkParallel $$(discover)
+main = checkParallel $$(discover) >>= bool exitFailure exitSuccess
```

Comment:
We were always running the tests, but we weren’t failing the execution of the test suite when the tests failed. I fixed this in `test-codegen.hs` too.